exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/exporter.py

```diff
@@ -282,3 +299,27 @@ def _parse_tags_str(tags_str):
             parsed_tags[key] = value
 
     return parsed_tags
+
+
+def _extract_tags_from_resource(resource):
+    """Parse tags from resource.labels, except service.name which
+    has special significance within datadog"""
+    tags = {}
+    service_name = None
+    if not resource and resource.labels:
```

Comment:
Did you mean `if not (resource and resource.labels):`

Reply:
👍 ah yup ty, updated